admin-manual/installation/installation.rst

```diff
@@ -167,27 +172,60 @@ Archivematica packages are hosted at packages.archivematica.org. This has been
 introduced to allow one central place to store packages for multiple OS's.
 Packages for both Ubuntu 14.04 and 16.04 are available.
 
+.. note:: Use of 14.04 is not recommended since 14.04 is currently in
+   maintenance updates only mode and will cease to be officially supported in
+   April 2019.
+
 1. Add Archivematica package sources
 
 Add packages.archivematica.org to your list of trusted repositories.
 
 Using 14.04 (Trusty):
 
+Run these three commands right now (**Delete this section when the final release
+packages are made**):
+
+.. code:: bash
+
+   sudo wget -O - https://packages.archivematica.org/1.7.x/key.asc | sudo apt-key add -
```

Comment:
I think we should add the urls for keys and repos to the conf.py, so we have one central place to do all the changes needed for the release.

Reply:
@scollazo I think I tried doing that originally and I had some trouble getting sphinx to do that interpolation in code blocks... I'll try again and report back

Reply:
Ok,if it's too hard forget about it. May be ading the full block might make it easier?

Reply:
ok, looks like it works if you use `.. parsed-literal::` instead of `.. code:: bash`. As long as we're not relying on code highlighting for bash commands in our docs, I think that'll be ok.